python/paddle/fluid/tests/unittests/autograd/test_primapi.py

```diff
@@ -170,7 +170,7 @@ def actual():
                     if isinstance(static_xs, typing.Sequence)
                     else self.fun(static_xs)
                 )
-                ys_grad = paddle.incubate.autograd.grad(ys, static_xs, static_v)
+                ys_grad = paddle.incubate.autograd.grad(ys)(static_xs, static_v)
```

Comment:
Using a new test instead of change the original one

Reply:
Done